core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/CollectInformation.scala

```diff
@@ -34,25 +34,25 @@ case class StageMetrics(numTasks: Int, duration: String)
 class CollectInformation(apps: Seq[ApplicationInfo]) extends Logging {
 
   def getAppInfo: Seq[AppInfoProfileResults] = {
-    val allRows = apps.map { app =>
+    val allRows = apps.filterNot(_.appInfo == null).map { app =>
       val a = app.appInfo
       AppInfoProfileResults(app.index, a.appName, a.appId,
         a.sparkUser,  a.startTime, a.endTime, a.duration,
         a.durationStr, a.sparkVersion, a.pluginEnabled)
     }
-    if (allRows.size > 0) {
+    if (allRows.nonEmpty) {
       allRows.sortBy(cols => (cols.appIndex))
     } else {
       Seq.empty
     }
   }
 
   def getAppLogPath: Seq[AppLogPathProfileResults] = {
-    val allRows = apps.map { app =>
+    val allRows = apps.filterNot(_.appInfo == null).map { app =>
```

Comment:
nit: Similar change as above

Reply:
addressed

Reply:
the comment was to add collect is how I read it

Reply:
I am also seeing the old version with `if else` and `filterNot`. I do see the date change!

Reply:
Apologies, my push went awry. Fixing here shortly.